docs/_guide/03-template-reference.md

```diff
@@ -320,87 +414,114 @@ import { guard } from 'lit-html/directives/guard';
 
 const template = html`
   <div>
-    ${guard(items, () => items.map(item => html`${item}`))}
+    ${guard([immutableItems], () => immutableItems.map(item => html`${item}`))}
   </div>
 `
 ```
 
-In this case, items are mapped over only when the array reference changes.
+In this case, the `immutableItems` array is mapped over only when the array reference changes.
 
-### until
-
-`until(...values)`
+### repeat 
 
-Renders one of a series of values, including Promises, to a Part.
+`repeat(items, keyfn, template)`
```

Comment:
We should show two signatures here, adding: `repeat(items, template)` without the keyFn